namesys/republisher/repub.go

```diff
@@ -55,15 +63,22 @@ func NewRepublisher(r routing.ValueStore, ds ds.Datastore, self ic.PrivKey, ks k
 }
 
 func (rp *Republisher) Run(proc goprocess.Process) {
-	tick := time.NewTicker(rp.Interval)
-	defer tick.Stop()
+	timer := time.NewTimer(InitialRebroadcastDelay)
+	defer timer.Stop()
+	if rp.Interval < InitialRebroadcastDelay {
```

Comment:
Doesn't this defeat the purpose of the initial delay?
I think we should simply tick with the InitialRebroadcastDelay unconditionally here.

Reply:
The initial delay is usually *much shorter* than the interval. If the user asks us to, e.g., publish every 10s, IMO, we should do that regardless of the initial delay (at the moment, not doing that would break the tests).

Alternatively, we could make the initial delay configurable (but that's one more knob).

Reply:
Yeah that's reasonable -- no need to have another knob for this.